### PR TITLE
Jb56649

### PR DESCRIPTION
**Pull request by martyone**

### CODE REVIEW

**Comment by martyone** [comment]
Just the last three commits belong to this PR

File: sdk-setup/src/mb2 [hunk 1]
```diff
@@ -731,6 +731,9 @@ GLOBAL OPTIONS
         packages, read the passphrase from the <file>. Only the first line will
         be read.
 
+    --package-timeline
```
Comment (vigejolla) [inline sdk-setup/src/mb2:732]: What's the story behind the name? Why not '--preserve-old-packages' ?

**Review by vigejolla — APPROVED** [review]
I'd like to get an answer to the one question, otherwise LGTM

File: sdk-setup/src/mb2 (hunk 1 above)
Comment (martyone) [reply sdk-setup/src/mb2:732]: In order to group well, the name should begin with "package" (`--foo-bar` is often represented as `foo.bar` on sfdk side), but `--package-preserve-old` would not read well. Hence the option to get a timeline of your packaged work, which fits well between the other `package.*` options.

**Pull request merged**